### PR TITLE
chore(developer): verify that package has at least a model or keyboard

### LINKED ISSUE #8791: chore(developer): package should have at least 1 keyboard or model

### PR DESCRIPTION
**Pull request by mcdurdin**
Fixes #8791.

Adds `ERROR_PackageMustContainAPackageOrAKeyboard` and corresponding unit tests. A couple of unit tests tweaked as their fixtures were no longer valid!

@keymanapp-test-bot skip

### CODE REVIEW

File: developer/src/kmc-package/src/compiler/messages.ts [hunk 1]
```diff
@@ -106,5 +106,9 @@ export class CompilerMessages {
   static Warn_DocFileDangerous = (o:{filename:string}) => m(this.WARN_DocFileDangerous,
     `Microsoft Word .doc or .docx files ('${o.filename}') are not portable. You should instead use HTML or PDF format.`);
   static WARN_DocFileDangerous = SevWarn | 0x0018;
+
+  static Error_PackageMustContainAPackageOrAKeyboard = () => m(this.ERROR_PackageMustContainAPackageOrAKeyboard,
```
Comment (darcywong00) [inline developer/src/kmc-package/src/compiler/messages.ts:110]: Would model be clearer?
```suggestion
  static Error_PackageMustContainAModelOrAKeyboard = () => m(this.ERROR_PackageMustContainAModelOrAKeyboard,
```

Comment (mcdurdin) [reply developer/src/kmc-package/src/compiler/messages.ts:110]: Good catch, oops my bad!

**Review by darcywong00 — APPROVED** [review]
lgtm

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 17.0.108-alpha](https://keyman.com/downloads/releases/alpha/17.0.108)

**Pull request merged**